### PR TITLE
Some minor fixes

### PR DESCRIPTION
**Pull request by Aureliona1**
I noticed a couple of minor issues with the script that would cause the code to break for people who imported the script rather than copying individual functions n stuff to their own map script. Also a couple of `let`s should have been `const`s.

It does look like a rather large change since I also just ran a formatter over the script and the imports, so lots of stuff got moved around.
If you would rather a non-formatted pr but with the same fixes then please let me know.

- Aurellis

### CODE REVIEW

File: src/mod.ts [hunk 1]
```diff
@@ -1,120 +1,133 @@
-import { Vec3, ColorType,Vec2, Geometry, activeDiffGet, Text, ModelScene, GeometryMaterial, AnimatedObjectInput} from "https://deno.land/x/remapper@3.1.2/src/mod.ts"
+import { activeDiffGet, AnimatedObjectInput, ColorType, Geometry, GeometryMaterial, KeyframesVec3, ModelScene, RawGeometryMaterial, Text, Vec2 } from "https://deno.land/x/remapper@3.1.2/src/mod.ts";
```
Comment (Aureliona1) [inline src/mod.ts:1]: Imports were missing `KeyframesVec3` and `RawGeometryMaterial`, also the formatter just put everything in alphabetical order.

File: src/mod.ts [hunk 2]
```diff
@@ -1,120 +1,133 @@
...
-type materialAlt = [string, RawGeometryMaterial][] // Name, Data
-export function appendMaterials(materials: materialAlt){ // Ex: materials = [["red",{color:[1,0,0]}],["blue",{color:[0,1,0]}]] 
-materials.forEach(mat => {
-    map.geoMaterials[mat[0]] = mat[1]
-})
+type materialAlt = [string, RawGeometryMaterial][]; // Name, Data
+export function appendMaterials(materials: materialAlt) {
+	// Ex: materials = [["red",{color:[1,0,0]}],["blue",{color:[0,1,0]}]]
+	materials.forEach(mat => {
+		activeDiffGet().geoMaterials[mat[0]] = mat[1];
```
Comment (Aureliona1) [inline src/mod.ts:34]: `activeDiffGet()` instead of `map.`

**Review by UGEcko — APPROVED** [review]
👍

**Review by UGEcko — APPROVED** [review]
👍

**Pull request merged**